### DISCUSSION TITLE
Welcome to Reshi Discussions!

### DISCUSSION THREAD

**Discussion by contrapasso3** [contrapasso3/Reshi#1] [Announcements]
## 👋 Hi there!
  We’re using Discussions as a place to improve this Obsidian theme. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas, suggestions, and issues.
  * Engage with other community members.

  To get started, comment below something you like about Reshi and something you'd add or change!

**Comment by nightlightouo** [comment]
Love this theme!

**Reply by contrapasso3** [reply]
Thanks so much!
